Assets/Scripts/Minigames/CowardMiniGame/RipcordBehavior.cs

```diff
@@ -21,9 +21,9 @@ public class RipcordBehavior : MonoBehaviour
 
     [Header("Ripcord Stats")]
     [SerializeField] private float _followPlayerSpeed;
-    [SerializeField] private Transform _maxPlayerDetectionPosition;
     [SerializeField] private float _returnSpeed;
-    
+    [SerializeField] private Transform _maxPlayerDetectionPosition;
```

Comment:
maybe I should have put this one under goal/scoring instead of ripcord stats initially (the cleanup looks nice btw)